pages/clubs.jsx

```diff
@@ -18,6 +18,29 @@ var ga = require('react-ga');
 
 var Illustration = require('../components/illustration.jsx');
 
+var PlaceSquare = React.createClass({
+  NUM_HASH_COLORS: 6,
```

Comment:
This has to be kept in-sync with the number of `.place-square.hash-color-*` rules in the LESS, which is a bit annoying.